### ISSUE TITLE
[feature-request] Add charge-lnd by accumulator

### ISSUE THREAD

**Issue by rootzoll** [raspiblitz/raspiblitz#4590]
### Discussed in https://github.com/raspiblitz/raspiblitz/discussions/3955

<div type='discussions-op-text'>

<sup>Originally posted by **DoktorWhat** June 20, 2021</sup>
https://github.com/accumulator/charge-lnd

Description:
This script matches your open Lightning channels against a number of customizable criteria and applies channel fees based on the matching policy.</div>

**Comment by rootzoll** [comment]
So the idea would be to crteate a bonus script like ./config.scripts/bonus.charge-lnd.sh that is based on bonus.template.sh (just without the need for a webui) that installs https://github.com/accumulator/charge-lnd in bitcoin user home with a config setup like:

```
echo "
[discourage-routing-out-of-balance]
chan.max_ratio = 0.1
chan.min_capacity = 250000
strategy = static
base_fee_msat = 2000
fee_ppm = 690

[encourage-routing-to-balance]
chan.min_ratio = 0.9
chan.min_capacity = 250000
strategy = static
base_fee_msat = 1000
fee_ppm = 21

[default-proportional]
chan.max_ratio = 0.9
chan.min_ratio = 0.1
chan.min_capacity = 250000
strategy = proportional
min_fee_ppm = 21
max_fee_ppm = 210
base_fee_msat = 2000
" >> /home/bitcoin/charge-lnd/charge.config

line="0 */6 * * * /home/bitcoin/charge-lnd/bin/charge-lnd -c /home/bitcoin/charge-lnd/charge.config"
(crontab -u $(whoami) -l; echo "$line" ) | crontab -u $(whoami) -
```

running it as a service.

**Comment by rootzoll** [comment]
PR #4594 ready to be tested & merged for v1.11.1rc1
